### DISCUSSION TITLE
Hourly Humidity Forecast - Accessible through API?

### DISCUSSION THREAD

**Discussion by devonschmidt** [weather-gov/api#202] [Q&A]
Hello, can you get hourly forecasted humidity information from an API call? The data seems available through the [website](https://f1.weather.gov/MapClick.php?w0=t&w1=td&w2=wc&w3=sfcwind&w4=sky&w5=pop&w6=rh&w7=rain&w8=thunder&w9=snow&w10=fzg&w11=sleet&AheadHour=38&Submit=Submit&&FcstType=digital&textField1=45.2768&textField2=-122.811&site=all), but is not returned using the api (I used the following call for example: 'https://api.weather.gov/gridpoints/MKX/88,60/forecast/hourly'). 

Your help is appreciated!

**Comment by JJKraw** [comment]
Just chop the "forecast/hourly/" off that URL and you get the raw data from the office. It has both dew point and relative humidity.

https://api.weather.gov/gridpoints/MKX/88,60

**Comment by devonschmidt** [comment]
Great, thanks!

**Comment by StephenClouse** [comment]
This would be the way to do it. The gridpoint data is in fact the raw data behind the tabular forecast/hourly weather graph.

We are looking at adding some of these additional fields to the forecast endpoints in a future release so there's less of a disconnect between the two.
